### ISSUE TITLE
Error While Integration with Vuetify

### ISSUE THREAD

**Issue by jefrydco** [davidroyer/nuxt-ssr-firebase#5]
Hi, I tried to integrate this project with Vuetify and it display an error in the console like this,
![error](https://user-images.githubusercontent.com/20434351/36381841-70723b74-15b9-11e8-9889-83224fb35821.png)

It seems every third party component can't be included in custom nuxt directory like this.

**Comment by Nayir** [comment]
Did you check the vuetify SSR documentation ? 
https://vuetifyjs.com/en/guides/server-side-rendering

**Comment by jefrydco** [comment]
Hi @Nayir 
That documentation just like any other documentation, doesn't mention specific needed for Vuetify to work. Beside that, my problem seems caused by the Nuxt configuration, not by Vuetify.

**Comment by AlexSwtlsk** [comment]
Did you find a solution to make it work with vuetify ?

**Comment by jefrydco** [comment]
Hi @AlexSwtlsk
I ended up with changin nuxt development mode to SPA, because I also have problem while deploying the project to firebase.

**Comment by AlexSwtlsk** [comment]
@jefrydco thanks for your answer. So, are you saying your repo https://github.com/jefrydco/nuxt-vuetify-firebase-ssr doesn't deploy on firebase or is not SSR anymore ?

**Comment by jefrydco** [comment]
@AlexSwtlsk Yes I'm, I'm currently use this repo https://github.com/jefrydco/nuxt-pwa-vuetify-starter to start of my project.

**Comment by AlexSwtlsk** [comment]
"yes" for which one of my questions ? 😄

**Comment by jefrydco** [comment]
:smile: I mean that I'm currently not use the repo on that link you attach. Instean I use the repo I mentioned earlier for starter.

**Comment by hoangbits** [comment]
@jefrydco How about deployment with firebase hosting progress?

**Comment by jefrydco** [comment]
@legiahoang I didn't try it again, but error I mentioned before is still exist
